aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -65,6 +64,28 @@ While supplemental attributes may need to be implemented and accessibility testi
 
 For guidance on setting up your development environment, see [Developer Guidance](/foundation/developer-guidance).
 
+## Testing for Accessibility
+
+Performing accessibility usability testing is not entirely linear and should be thought of from the beginning. The following are steps you should look to incorporate into your workflow to better test and ensure accessibility across the board:
```

Comment:
I'm not exactly sure what is meant by "should be thought of from the beginning" -- meaning should be incorporate from the beginning of development?

Reply:
I was confused by the first sentence as well. It seemed to be trying to speak to two different subjects, but was presented as they are related or aspects of the same thing.

Reply:
Hmmm I see your point here, I can rephrase to align more with incorporating accessibility from the beginning of the design process (since that's the first process before development).